moler/cmd/at/set_cell_id_nr.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+AT+C5GREG
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Jakub Kochaniak'
+__copyright__ = 'Copyright (C) 2023, Nokia'
+__email__ = 'jakub.kochaniak@nokia.com'
+
+from moler.cmd.at.genericat import GenericAtCommand
+from moler.exceptions import WrongUsage
+
+
+class SetCellIdNr(GenericAtCommand):
+    """
+    Command to set NR cell registration.
+
+    Possible options for selected_mode, but verify with latest 3gpp specification 27.007:
+    0: 'disable network',
+    1: 'enable network',
+    2: 'enable network, location information unsolicited result code',
+    3: 'enable network, location information and 5GMM cause value information unsolicited result code',
+    4: 'enable network, location information, cause value information, CAG status unsolicited result code',
+    5: 'enable network, location information, cause value information, CAG status and CAG cell status '
+       'unsolicited result code'
+    """
+
+    def __init__(self, selected_mode, connection=None, prompt=None, newline_chars=None, runner=None):
```

Comment:
Please add docstring

Reply:
Moved docstring content to every constructor in this PR files.